Tests/SentryTests/Transaction/SentrySpanTests.swift

```diff
@@ -451,4 +468,79 @@ class SentrySpanTests: XCTestCase {
         XCTAssertEqual(nameForSentrySpanStatus(.outOfRange), kSentrySpanStatusNameOutOfRange)
         XCTAssertEqual(nameForSentrySpanStatus(.dataLoss), kSentrySpanStatusNameDataLoss)
     }
+    
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+    func testAddSlowFrozenFramesToData() {
+        let (displayLinkWrapper, framesTracker) = givenFramesTracker()
+        
+        let sut = SentrySpan(context: SpanContext(operation: "TEST"), framesTracker: framesTracker)
+        
+        let slow = 2
+        let frozen = 1
+        let normal = 100
+        displayLinkWrapper.givenFrames(slow, frozen, normal)
+        
+        sut.finish()
+        
+        expect(sut.data["frames.total"] as? NSNumber) == NSNumber(value: slow + frozen + normal)
+        expect(sut.data["frames.slow"] as? NSNumber) == NSNumber(value: slow)
+        expect(sut.data["frames.frozen"] as? NSNumber) == NSNumber(value: frozen)
+    }
```

Comment:
I didn't add too many tests because we use the same logic as for SentryTracer. Anyways, adding your test case adds value, so I added it. Thanks for pointing it out 😃